### PR TITLE
010

### PR DESCRIPTION
**Pull request by TiTANWASTAKEN**

### CODE REVIEW

File: basic login/login.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<html>
+<head>
+  <title></title>
```
Comment (SaketJNU) [inline basic login/login.html:3]: Put Title name here as "Initial Login Page"

File: basic login/style.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	margin: 20px auto;
+	text-align: center;
+	border: 3px solid #0367fd;
+	padding :14px 10px;
+	width: 220px;
+	outline:none;
+	color:white;
+	border-radius:24px;
+	transition:0.25px;
+
```
Comment (SaketJNU) [inline basic login/style.css:33]: Remove this black space 33 line

**Review by SaketJNU — CHANGES_REQUESTED** [review]
Complete it asap

File: basic login/style.css (hunk 2 above)
Comment (TiTANWASTAKEN) [reply basic login/style.css:33]: updation done.

**Review by SaketJNU — APPROVED** [review]

**Comment by SaketJNU** [comment]
1st PR done

**Pull request merged**